lib/FieldLocation/FieldLocationContainer.js

```diff
@@ -67,11 +71,17 @@ FieldLocationFinalContainer.propTypes = {
   onChange: PropTypes.func.isRequired,
   prepopulatedLocationsIds: PropTypes.arrayOf(PropTypes.string).isRequired,
   required: PropTypes.bool,
+  isReduxField: PropTypes.bool,
+  validate: PropTypes.oneOfType([
+    PropTypes.func,
+    PropTypes.arrayOf(PropTypes.func),
+  ]),
 };
 
-FieldLocationFinalContainer.defaultProps = {
```

Comment:
I don't think that naming inside implementation has any importance. It could be leaved as is, in case of https://github.com/folio-org/ui-receiving/pull/168#discussion_r436990652